master/internal/api_workspace.go

```diff
@@ -519,12 +558,34 @@ func (a *apiServer) deleteWorkspaceNamespaceBinding(ctx context.Context,
 			if err != nil {
 				return err
 			}
+			deletedClusters := []string{}
 			for _, v := range deletedBindings {
+				deletedClusters = append(deletedClusters, v.ClusterName)
 				err = a.m.rm.RemoveEmptyNamespace(v.Namespace, v.ClusterName)
 				if err != nil {
 					return err
 				}
 			}
+			if len(deletedClusters) < len(clusterNames) {
+				switch {
+				case len(clusterNames) == 1 && (clusterNames[0] != ""):
+					return fmt.Errorf("tried to delete default binding for cluster %v", clusterNames[0])
+				case len(clusterNames) == 1:
+					return fmt.Errorf("tried to delete default binding for cluster")
+				default:
+					remainingClusters := ""
+					log.Print(deletedBindings)
```

Comment:
Is this for debugging or for printing regular info? I think we should use the standard logging levels (maybe something like `log.Infof` or `log.Debugf`) 

(credit @keita-determined for helping me spot this :)

Reply:
Oops, I think I it was for debugging, i'll remove it!